data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -617,6 +617,46 @@
         {CLEAR_VARIABLE delfador_moves}
     [/event]
 
+    # Introductory dialog to give the player a hint that she may lose units to
+    # {EXPAND_LAVA}.
+    [event]
+        name=turn 2
+        # The noise was played by {EXPAND_LAVA}.
+        [message]
+            id=Konrad
+            message=_ "That noise was made by no troll! Can you tell its source?"
+        [/message]
+        [if]
+            [have_unit]
+                race=dwarf
+            [/have_unit]
+            [then]
+                [message]
+                    race=dwarf
+                    message=_ "Lord Konrad, yon rumble was made by nae enemy o’ us. ’Tis the echoes of the very bedrock witherin’ and crumbling beneath us that we ’eard. Are ye feeling naught heat and loess redoubling around us? I beseech that we not tarry ’ere any more than we ha’ to, lest the forming abysses claim us."
+                [/message]
+            [/then]
+            [elseif]
+                [have_unit]
+                    type=Arch Mage,Great Mage,Silver Mage
```

Comment:
Only the high-level mages, huh?

Reply:
I'm not really sure. Your reasoning makes sense, at least.

Reply:
Documented in a comment.